nixos/modules/services/networking/tailscale-cert.nix

```diff
@@ -0,0 +1,143 @@
+{ config, options, lib, pkgs, ... }:
+
+let
+  cfg = config.services.tailscale-cert;
+  opt = options.services.tailscale-cert;
+in
+{
+  options.services.tailscale-cert = {
+    enable = lib.mkEnableOption "the Tailscale cert service";
+
+    package = lib.mkPackageOption pkgs "tailscale" { };
+
+    domain = lib.mkOption {
+      type = lib.types.str;
+      description = "Tailnet domain for which to request the certificate";
+      example = "monitoring.tailnet0123.ts.net";
+    };
+
+    dataDir = lib.mkOption {
+      type = lib.types.str;
+      default = "tailscale-cert";
+      description = ''
+        Directory name under /var/lib/ where to store certificates and keys.
+      '';
+    };
+
+    certFile = lib.mkOption {
+      type = lib.types.str;
+      default = "${cfg.domain}.crt";
+      defaultText = lib.literalExpression "config.${opt.domain}.crt";
+      description = "File name for cert under <option>services.tailscale-cert.dataDir</option>";
```

Comment:
We can still use <option> after the switch to markdown in the documentation? TIL :)

Reply:
Good catch, fixed :)